### PR TITLE
added color and size as props to Text Component

### PR DESCRIPTION
**Pull request by AnthoniaOkafor**

### CODE REVIEW

**Comment by AnthoniaOkafor** [comment]
Hello @meekg33k . I have resolved conflicts with this branch

File: src/components/Text/index.js [hunk 1]
```diff
@@ -1,10 +1,16 @@
-import React from "react"
 
-const Text = (props) => {
-  const { text } = props
+import React from "react";
+import './Text.css';
 
+//<Text text='Sample text' color='green' size="small" />
```
Comment (meekg33k) [inline src/components/Text/index.js]: @AnthoniaOkafor, can you please remove all commented code from your PR?

File: src/App.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const App = () => {
   return (
     <div>
       <Routes />
-      <Text text="It is a text" />
+      <Text color="green" size="medium" text="It is a text" />
```
Comment (meekg33k) [inline src/App.js]: @AnthoniaOkafor, we should be using the correct color specified in the mockup design not green

Comment (AnthoniaOkafor) [reply src/App.js]: ok @meekg33k

File: src/components/Text/index.js (hunk 1 above)
Comment (AnthoniaOkafor) [reply src/components/Text/index.js]: I will @meekg33k

**Pull request merged**